### DISCUSSION TITLE
how to enable toomanyactiveblocksinclaim

### DISCUSSION THREAD

**Discussion by notjelly** [GriefPrevention/GriefPrevention#1118] [Q&A] [answered]
The answer to your question may exist in the wiki, go check that out first! 

Wiki: https://github.com/TechFortress/GriefPrevention/wiki

Otherwise, if you have a spigot account, you may have better luck posting in the discussion thread so people who follow the thread may see your question. https://www.spigotmc.org/threads/griefprevention.35615/page-311

Still no luck? Really? Then go ahead and ask here.

Please remember to provide as much detail as you can - smart questions are
much more likely to be answered.  See "Asking questions the smart way":
http://www.catb.org/~esr/faqs/smart-questions.html

As an open-source project, we're all volunteers with limited time to spend,
so providing enough info (including any error messages etc) up-front makes
it more likely we can provide you an answer with limited time.

**Comment by Jikoo** [comment, marked as answer]
Duplicate of #1119
